packages/flutter/test/material/scaffold_test.dart

```diff
@@ -15,20 +15,24 @@ void main() {
     bool isDrawerOpen = false;
     bool isEndDrawerOpen = false;
 
+    DrawerCallback drawerCallback = (isOpen) {
+      isDrawerOpen = isOpen;
+    };
+
+    DrawerCallback endDrawerCallback = (isOpen) {
+      isEndDrawerOpen = isOpen;
+    };
+
     await tester.pumpWidget(MaterialApp(
       home: Scaffold(
           drawer: Container(
             color: Colors.blue,
           ),
-          drawerCallback: (isOpen) {
-            isDrawerOpen = isOpen;
-          },
+          drawerCallback: drawerCallback,
```

Comment:
@chunhtai Could you please help me resolve warning 'Specify type annotations' in his line?